backend/amp-live-list.go

```diff
@@ -146,7 +154,7 @@ func createMetadata(r *http.Request) []BlogPosting {
 	for i := 0; i < len(blogs); i++ {
 		result = append(result, BlogPosting{"BlogPosting",
 			blogs[i].Heading,
-			buildSourceOrigin(r.Host) + "/" + CATEGORY_SAMPLE_TEMPLATES + "/live_blog/#" + blogs[i].ID,
+			buildSourceOrigin(r.Host) + "/" + CATEGORY_SAMPLE_TEMPLATES + "/live_blog/#" + fmt.Sprintf("%s%d", BLOG_ID_PREFIX, i+1),
```

Comment:
I find it weird to combine concatenation via `+` and `fmt.Sprintf`. Please use only one.